src/en/isaac-penington.yml

```diff
@@ -25,6 +25,181 @@ documents:
             type: epub
           -
             type: mobi
+        audio:
+          reader: Jason Henderson
+          parts:
+            -
+              title: Part 1
+              external_id_hq: 236087816
+              external_id_lq: 236087828
+              filesize_hq: 48529714
+              filesize_lq: 17356902
+              seconds: 2893
+              chapters:
+                - 1
```

Comment:
These should start with `0` if it corresponds to the very first chapter.  Take a look at the `william-penn.yml`.